pkg/storage/swift/swift_test.go

```diff
@@ -144,6 +147,25 @@ func handleAuthentication(t *testing.T, endpointType string) {
 	})
 }
 
+func fakeInfrastructureLister(cloudName string) configlisters.InfrastructureLister {
```

Comment:
Can this go here?
https://github.com/openshift/cluster-image-registry-operator/tree/master/test/framework/mock/listers

Also, should some of these tests in this file really be in e2e?